src/main/scala/dev/atedeg/Table.scala

```diff
@@ -0,0 +1,71 @@
+package dev.atedeg
+
+import java.util.Locale.UK
+
+import scala.util.Try
+
+import dev.atedeg.HtmlParsing.{ extractClassLike, extractNonClassLike }
+
+import better.files.File
+import cats.implicits.*
+import net.steppschuh.markdowngenerator.table.Table.ALIGN_LEFT
+import net.steppschuh.markdowngenerator.table.Table.Builder
+
+import Extensions.*
+
+final case class Row(term: String, definition: String)
+object Row {
+  private def normalizeName(name: String): String =
+    name.split("(?=\\p{Upper})").toList.map(_.toLowerCase(UK)).mkString(" ").capitalize
+
+  def buildRow(termDefinition: (String, String)): Row =
+    Row(normalizeName(termDefinition._1), termDefinition._2)
+}
+
+final case class Table(title: String, termName: String, definitionName: String, rows: List[Row]) {
```

Comment:
Could you make this constructor private too?

Reply:
No since it's the only way we have to build a table